src/gui_common/charts/line/LineChart.tscn

```diff
@@ -1,26 +1,10 @@
-[gd_scene load_steps=12 format=2]
+[gd_scene load_steps=8 format=2]
 
-[ext_resource path="res://assets/fonts/Jura-DemiBold.ttf" type="DynamicFontData" id=1]
-[ext_resource path="res://assets/fonts/Lato-Regular.ttf" type="DynamicFontData" id=2]
 [ext_resource path="res://src/gui_common/thrive_theme.tres" type="Theme" id=3]
 [ext_resource path="res://src/gui_common/charts/line/LineChart.cs" type="Script" id=4]
 [ext_resource path="res://assets/textures/gui/bevel/hSeparatorLeftSide.png" type="Texture" id=5]
 [ext_resource path="res://assets/textures/gui/bevel/vSeparatorUp.png" type="Texture" id=6]
-
-[sub_resource type="DynamicFont" id=1]
-size = 14
-use_mipmaps = true
-use_filter = true
-font_data = ExtResource( 1 )
-
-[sub_resource type="DynamicFont" id=2]
-size = 12
-use_mipmaps = true
-use_filter = true
-font_data = ExtResource( 2 )
-
-[sub_resource type="Theme" id=3]
```

Comment:
These themes are needed to automatically set a specific font for all labels inside a container that were added runtime, otherwise it uses thrive_theme's default font (if set on the root control), which doesn't necessarily has the right font and size for them. For example, the labels on the mouse hover panel uses thrive theme's default font (which is too big and uses jura, the labels should use lato font):
![2020-12-04_23 36 02 8457](https://user-images.githubusercontent.com/54026083/101189503-b9c44800-3689-11eb-9edc-6b36bd32c65d.png)
Though, I suppose it could also be done by setting the font for a label in the code after instantiating it.

Reply:
I nuked the separate themes that contained DynamicFont definitions in them in order to get rid of them. I did quickly try to visually confirm that nothing changed. Could I bother you to create a new theme / theme file and apply it to those places (I assume you know better where the themes are needed and can probably finish it much quicker).

Reply:
I'll try do that.

Reply:
Thanks. Sorry to cause you extra work like this.